support-frontend/assets/pages/contributions-landing/contributionsLandingActions.js

```diff
@@ -351,7 +358,8 @@ const regularPaymentRequestFromAuthorisation = (
     city: null, // required go cardless field
     state: state.page.form.formData.state,
     postCode: null, // required go cardless field
-    country: state.common.internationalisation.countryId,
+    country:
+      state.page.form.formData.country ? state.page.form.formData.country : state.common.internationalisation.countryId,
```

Comment:
When would these two ever be different?

Reply:
In all other instances the same processing/source-selection happens for state as does country, except for here...

Reply:
If I am on `/ca/contribute` and use the single pay button with a card that has a US based address

`state.common.internationalisation.countryId` will be CA
`state.page.form.formData.country` will be US (taken from the Stripe auth res)

Reply:
Currently:
`state.common.internationalisation` has country but not state
`state.page.form` has state but not country
When we POST we use both.

This branch adds a `country` field to `state.page.form`.
It may be preferable to instead update the `country` in `state.common.internationalisation`. But I'm not sure it's safe to go mutating that (there it is tied up with currency as well).
Let's discuss with @paulbrown1982 when we're all around

Reply:
agreed `state.page.form.formData.country` to be renamed `state.page.form.formData.billingCountry`